control-plane/pkg/reconciler/kafka/topic.go

```diff
@@ -31,6 +39,38 @@ type TopicConfig struct {
 	BootstrapServers []string
 }
 
+func TopicConfigFromConfigMap(logger *zap.Logger, cm *corev1.ConfigMap) (*TopicConfig, error) {
```

Comment:
can we pass a `Validator` here and reconcilers can just pass their own validators?

Reply:
I thought about that. But, to me, returning the topicConfig and then explicitly do the validation in the reconciler seemed cleaner.